### PR TITLE
BCR contribution policy

### PR DESCRIPTION
**Pull request by radvani13**
Adding a BCR contribution policy

### CODE REVIEW

File: docs/bcr-contribution-guide.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:1]:
```suggestion
```

File: docs/bcr-contribution-guide.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Bazel Central Registry (BCR) contribution policy contribution policies
+Contributors should follow these policies when reviewing and accepting modules to the Bazel Central Registry:
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:3]:
```suggestion
# Bazel Central Registry (BCR) contribution policy contribution policies

Contributors should follow these policies when reviewing and accepting modules to the Bazel Central Registry:
```

File: docs/bcr-contribution-guide.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+-   All changes to the BCR repository must be submitted and reviewed as a pull request.
+-   Add-only: existing module versions must not be modified to ensure reproducible builds.
+-   Every module version must have a [presubmit.yml](https://docs.google.com/document/d/1moQfNcEIttsk6vYanNKIy3ZuK53hQUFq1b1r0rmsYVg/edit#heading=h.e6t527rxhw5i) file.
+-   BCR modules can only depend on modules already in the BCR.
+
+## BCR maintenance policies
+Registry maintainers and module maintainers maintain the BCR. BCR maintainers are members of the community who have restricted ability to accept Bazel-built reusable packages into the BCR.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:11]:
```suggestion
## BCR maintenance policies

Registry maintainers and module maintainers maintain the BCR. BCR maintainers are members of the community who have restricted ability to accept Bazel-built reusable packages into the BCR.
```

File: docs/bcr-contribution-guide.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Become a BCR maintainer
+- We recommend contributing and maintaining a Bazel module before becoming a BCR Maintainer.
+- All maintainer requests will be reviewed by the [Rules Authors SIG](https://github.com/bazel-contrib/SIG-rules-authors) and the Bazel team before approval.
+
+### Responsibilities of a BCR maintainer
+- Review and accept contributions made by the community that align with the BCR contribution policy. Maintainers can merge any PR they have carefully reviewed and is passing CI. BCR maintainer review is required when:
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:18]:
```suggestion
### Responsibilities of a BCR maintainer

- Review and accept contributions made by the community that align with the BCR contribution policy. Maintainers can merge any PR they have carefully reviewed and is passing CI. BCR maintainer review is required when:
```

File: docs/bcr-contribution-guide.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  - Raise a PR to add a new module for the first time to the BCR.
+  - Raise a PR to change a presubmit.yml file for a module.
+- Assess the health of the BCR by monitoring the BCR testing and serving infrastructure.
+- Identify and appoint module maintainers.
+- Triage issues in the BCR github repository.
+
+## Become a module maintainer
+Every module must have a list of maintainers provided in its metadata.json file. The first contributor to a Bazel module is likely be appointed as the module maintainer. Anyone can apply to be a maintainer of some specific modules. Library authors / rule authors / Bazel users who have special interest on some projects can be module maintainers.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:26]:
```suggestion
## Become a module maintainer

Every module must have a list of maintainers provided in its metadata.json file. The first contributor to a Bazel module is likely be appointed as the module maintainer. Anyone can apply to be a maintainer of some specific modules. Library authors / rule authors / Bazel users who have special interest on some projects can be module maintainers.
```

File: docs/bcr-contribution-guide.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### Responsibilities of a module maintainer
+-   Review and approve changes related to the designated Bazel module.
+-   Update new versions for the designated Bazel module.
+
+## Contacting the repository or module maintainers
+Whenever possible, conduct all communication in public on GitHub. Contact maintainers via GitHub or Slack. Contact maintainers over slack group [bzlmod](https://bazelbuild.slack.com/archives/C014RARENH0) when contacting over GiHub is not possible.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:33]:
```suggestion
## Contacting the repository or module maintainers

Whenever possible, conduct all communication in public on GitHub. Contact maintainers via GitHub or Slack. Contact maintainers over slack group [bzlmod](https://bazelbuild.slack.com/archives/C014RARENH0) when contacting over GiHub is not possible.
```

File: docs/bcr-contribution-guide.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+### Maintenance timeline expectations
+Maintainers are community-driven volunteers. Hence, most PRs may be addressed in 3 business days. If a PR is not marked with a label "awaiting user response", then it should be addressed by a register maintainer within 3 days.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:36]:
```suggestion
### Maintenance timeline expectations

Maintainers are community-driven volunteers. Hence, most PRs may be addressed in 3 business days. If a PR is not marked with a label "awaiting user response", then it should be addressed by a register maintainer within 3 days.
```

File: docs/bcr-contribution-guide.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+
+# Bazel Central Registry (BCR) contribution policy contribution policies
+Contributors should follow these policies when reviewing and accepting modules to the Bazel Central Registry:
+
+-   All changes to the BCR repository must be submitted and reviewed as a pull request.
+-   Add-only: existing module versions must not be modified to ensure reproducible builds.
+-   Every module version must have a [presubmit.yml](https://docs.google.com/document/d/1moQfNcEIttsk6vYanNKIy3ZuK53hQUFq1b1r0rmsYVg/edit#heading=h.e6t527rxhw5i) file.
+-   BCR modules can only depend on modules already in the BCR.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:8]:
```suggestion
- All changes to the BCR repository must be submitted and reviewed as a pull request.
- Add-only: existing module versions must not be modified to ensure reproducible builds.
- Every module version must have a [presubmit.yml](https://docs.google.com/document/d/1moQfNcEIttsk6vYanNKIy3ZuK53hQUFq1b1r0rmsYVg/edit#heading=h.e6t527rxhw5i) file.
-   BCR modules can only depend on modules already in the BCR.
```

File: docs/bcr-contribution-guide.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+  - Raise a PR to add a new module for the first time to the BCR.
+  - Raise a PR to change a presubmit.yml file for a module.
+- Assess the health of the BCR by monitoring the BCR testing and serving infrastructure.
+- Identify and appoint module maintainers.
+- Triage issues in the BCR github repository.
+
+## Become a module maintainer
+Every module must have a list of maintainers provided in its metadata.json file. The first contributor to a Bazel module is likely be appointed as the module maintainer. Anyone can apply to be a maintainer of some specific modules. Library authors / rule authors / Bazel users who have special interest on some projects can be module maintainers.
+
+### Responsibilities of a module maintainer
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:28]:
```suggestion
### Responsibilities of a module maintainer

```

File: docs/bcr-contribution-guide.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+-   Review and approve changes related to the designated Bazel module.
+-   Update new versions for the designated Bazel module.
```
Comment (brentleyjones) [inline docs/bcr-contribution-guide.md:30]:
```suggestion
- Review and approve changes related to the designated Bazel module.
- Update new versions for the designated Bazel module.
```

**Review by brentleyjones — COMMENTED** [review]
Just some Markdown formatting suggestions, to better match the spec.

**Review by alexeagle — APPROVED** [review]

**Review by fmeum — APPROVED** [review]

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

**Comment by radvani13** [comment]
Hi All, I think I've got everything, but I am not very good with git - so please let me know

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**